easybuild/easyconfigs/j/Jupyter-bundle/Jupyter-bundle-20240522-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Bundle'
+
+name = 'Jupyter-bundle'
+version = '20240522'
+
+homepage = "https://jupyter.org/"
+
+description = """
+ This bundle collects a range of Jupyter interfaces (Lab, Notebook and nbclassic),
+  extensions (Jupyter Server Proxy, Jupyter Resource Monitor, Jupyter Lmod) and
+  the JupyterHub.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+dependencies = [
+    ('JupyterHub', '4.1.5'),
+    ('JupyterLab', '4.2.0'),
+    ('JupyterNotebook', '7.2.0'),
+    ('nbclassic', '1.0.0'),
+    ('jupyter-server-proxy', '4.1.2'),
+    # ('jupyterlmod', '5.0.0'), -- not ready yet, waiting for https://github.com/cmd-ntrf/jupyter-lmod/pull/70
```

Comment:
@lexming assuming the tests are successful and this is merged as is, you'll come back to this later? or should we simply hold off merging this for now?

Reply:
We can merge the bundle without `jupyterlmod` for the moment. I left the comment so that people know why it is not there compared to the previous bundle. I'll make another PR whenever it is released.